### ISSUE TITLE
"nether_wastes" biome seems broken/missing

### ISSUE THREAD

**Issue by BusterBeachside** [Amulet-Team/Amulet-Map-Editor#519]
MC version: Java 1.18.1

Amulet version: V0.8.17b0

![image](https://user-images.githubusercontent.com/83925088/147393613-a69a93fd-cab0-46bf-a974-29313749b67f.png)

I'm trying to create a "one chunk" world. As a part of this, I wanted to set all of the empty chunks nearby my one full one to the "nether wastes" biome. However, while other Nether biomes, such as Warped Forest and Basalt Delta, appear in the list and seem to be working, the Nether Wastes simply show up as "nether". When attempting to set this as the biome, the world defaults those chunks to a Plains biome the next time the world is loaded in-game, which causes unexpected overworld mobs to be able to spawn (Such as creepers and spiders). 

Expected behavior would of course be to properly set the biome as "Minecraft:nether_wastes" and to have the biome listed under that name within Amulet.

First time submitting an issue on Github, let me know if I've missed anything. :)

**Comment by Jason-Silla** [comment]
I’ve been having this same issue. Hopefully the devs fix this really soon!!!

**Comment by BusterBeachside** [comment]
UPDATE: This is still an issue in V0.8.17b1.dev202201130714. However, I've found a workaround for this issue.
When in the "Set Biome" operation menu, "nether_wastes" still doesn't show up in the menu, and setting the biome to "nether" results in the biome reverting to Plains upon loading the world in 1.18. However, by typing or pasting "nether_wastes" in the "Biome name" field (The one with the search icon) and then running the operation, the correct biome actually gets set once loaded in-game! For the record, I ran the operation twice, first in "Selected Box" mode and then again in "Selected Column" mode. I'm not sure if that is in any way a required part to work around the issue or if simply pasting in the correct biome name and running the operation once would suffice.

Expected behavior remains that "nether_wastes" would appear in the list of biomes in place of "nether".
